Demo/Demo/SwiftUIComponents/PayPalWebViews/PayPalTransactionsView.swift

```diff
@@ -1,7 +1,7 @@
 import SwiftUI
 import PaymentButtons
 
-struct PayPalTransactionButtonsView: View {
+struct PayPalTransactionsView: View {
```

Comment:
```suggestion
struct PayPalTransactionView: View {
```

Reply:
There are three options of transactions on that view PayPal, PayPal Credit, Pay Later. That's why I put Transactions. What do you think?

Reply:
I think the singular is fine even though it's used in multiple places there will ever only be 1 transaction